node/src/components/storage.rs

```diff
@@ -863,7 +863,10 @@ impl Storage {
                     &block_header,
                     false,
                 )? {
-                    error!("Could not insert block header: {}", block_header);
+                    error!(
+                        "Could not insert block header (maybe already inserted?): {}",
+                        block_header
+                    );
```

Comment:
```rust
                    error!(
                    ?block_header
                        "Could not insert block header (maybe already inserted?)",
                    );
```

otherwise it's not going to be indexed/displayed properly in Kibana.

Reply:
Should be fixed in 752c1770d